test/e2e/harbor.go

```diff
@@ -28,3 +28,15 @@ func runCuratedPackageHarborInstallSimpleFlowLocalStorageProvisioner(test *frame
 		test.VerifyHarborPackageInstalled(packagePrefix, installNs)
 	})
 }
+
+func runCuratedPackageHarborInstallTinkerbellSimpleFlow(test *framework.ClusterE2ETest) {
+	test.GenerateClusterConfig()
+	test.GenerateHardwareConfig()
+	test.PowerOnHardware()
+	test.CreateCluster(framework.WithControlPlaneWaitTimeout("20m"))
+	test.ValidateControlPlaneNodes(framework.ValidateControlPlaneNoTaints, framework.ValidateControlPlaneLabels)
+	runCuratedPackageHarborInstallSimpleFlowLocalStorageProvisioner(test)
```

Comment:
I dont think we can reuse this in this manner. This has a test.WithCluster() which i think internally also does a CreateCluster.

We might have to take what this function does and do it here.